rts/motoko-rts/src/gc/mark_compact/bitmap.rs

```diff
@@ -117,8 +128,10 @@ impl BitmapIter {
             if self.current_word_idx == self.size {
                 return BITMAP_ITER_END;
             }
-            self.current_word =
-                unsafe { *(BITMAP_PTR as *const u64).add(self.current_word_idx as usize) };
+            self.current_word = unsafe {
+                *(BITMAP_PTR.add(BITMAP_COMPENSATION) as *const u64)
+                    .add(self.current_word_idx as usize)
```

Comment:
`add(BITMAP_COMPENSATION)` part can be avoided if we cache the blob payload location, as before. See my comment at the beginning of this file.

Reply:
The iteration code is bigly reworked, should be moderately more efficient too.